src/os/bluestore/BlueStore.cc

```diff
@@ -8014,8 +8014,10 @@ int BlueStore::_fsck_on_open(BlueStore::FSCKDepth depth, bool repair)
   _fsck_collections(&errors);
   used_blocks.resize(fm->get_alloc_units());
 
-  int r = bluefs->get_block_extents(bluefs_layout.shared_bdev, &bluefs_extents);
-  ceph_assert(r == 0);
+  if(bluefs) {
+    int r = bluefs->get_block_extents(bluefs_layout.shared_bdev, &bluefs_extents);
```

Comment:
Please add space between "if" and "(".

Reply:
@aclamk I have added space between "if" and "(". (I remember that I reply the change but it seems not  success)